app/models/game.rb

```diff
@@ -1,7 +1,35 @@
 class Game < ApplicationRecord
-  has_many :fear_ratings
-  has_many :reviews
+  #one-many
+  has_many :fear_ratings, dependent: :destroy
+  has_many :reviews, dependent: :destroy
+  has_many :screenshots, dependent: :destroy
+  has_many :videos, dependent: :destroy
+
+
+  #many-many
   has_many :user_games, dependent: :destroy
   has_many :users, through: :user_games
-  validates_presence_of :title
+
+  has_many :game_categories, dependent: :destroy
+  has_many :categories, through: :game_categories
+
+  has_many :game_genres, dependent: :destroy
+  has_many :genres, through: :game_genres
+
+  has_many :game_keywords, dependent: :destroy
+  has_many :keywords, through: :game_keywords
+
+  has_many :game_platforms, dependent: :destroy
+  has_many :platforms, through: :game_platforms
+
+  has_many :game_similars, dependent: :destroy
+  has_many :similars, through: :game_similars
+
+  has_many :game_themes, dependent: :destroy
+  has_many :themes, through: :game_themes
+
```

Comment:
oy vay

Reply:
Thanks for tackling this for the team! Great work

Reply:
relationships look good. I think once we dive into the API more, we could probably eliminate between categories, genres, and themes and choose one since they sound very familiar.